<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LayerListWindow</class>
 <widget class="QWidget" name="LayerListWindow">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>491</width>
    <height>419</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Layer(s)</string>
  </property>
  <widget class="QListView" name="lstLayers">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>471</width>
     <height>301</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="btnFetch">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>30</y>
     <width>98</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Fetch</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtSearch">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>361</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="btnDone">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>380</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Done</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
